### PR TITLE
List messages 

### PR DESCRIPTION
**Pull request by karelrochelt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,12 +54,37 @@ Parameters:
   - **channelId** - ID of your Amio Webchat channel
   - **localStorageSessionName** - Allows to customize the name of the Local Storage field that holds a session ID
```
Comment (kuceram) [inline README.md]: Tady chybi carky na konci vety.

File: README.md [hunk 2]
```diff
@@ -52,12 +54,37 @@ Parameters:
   - **channelId** - ID of your Amio Webchat channel
```
Comment (kuceram) [inline README.md]: Tady chybi carky na konci vety.

File: README.md (hunk 1 above)
Comment (kuceram) [reply README.md]: Zminil bych ze je to optional.

File: README.md [hunk 3]
```diff
@@ -52,12 +54,37 @@ Parameters:
...
   - **localStorageSessionName** - Allows to customize the name of the Local Storage field that holds a session ID
 
+```js
+AmioWebchatClient.connect({
+  channelId: '12345678'
```
Comment (kuceram) [inline README.md]: tady bud pouyij placeholder `{CHANNEL_ID}`, a nebo uved realne cislo `6495613231087502282`

File: README.md [hunk 4]
```diff
@@ -52,12 +54,37 @@ Parameters:
...
+.then(() => {
+  console.log('Message sent successfully')
+})
+.catch(err => {
+  console.log('Error while sending message:', err)
+})
+```
+
 ### sendTextMessage(text)
 Sends a text message. This is just a handy shortcut for `sendMessage({type: 'text', payload: '...'})`
```
Comment (kuceram) [inline README.md]: nechybi example?

File: README.md [hunk 5]
```diff
@@ -70,6 +97,79 @@ Sends an event indicating that all received messages were read by the receiver.
...
+### listMessages(max, cursor)
+Loads messages from message history. Can be called multiple times to move further back in history.
+
+Parameters:
+- **max** - Number of messages to load. Should be between 1 and 100.
+- **cursor** - Reference point (message id) indicating where to start loading messages in history. If set to `null`, messages will be read from the beginning (newest first).
+
+Response format:
+- **messages** - Array of messages, sorted from newest to oldest.
+- **cursor.next** - Cursor that should be used in the next call of listMessages().
```
Comment (kuceram) [inline README.md]: Cursor pointing the very next message. Use this cursor in the next call of `listMessages()`.

Comment (zatziky) [reply README.md]: Ještě bych upravil "Curose pointing to subsequent messages. ..."

**Pull request merged**